system/security/LdapSecurity/ldaputils.cpp

```diff
@@ -105,7 +105,41 @@ LDAP* LdapUtils::LdapInit(const char* protocol, const char* host, int port, int
             return nullptr;
         }
 #endif
+
+    }
+
+    //Set TLS KeepAlive options
+    int kaTime;
+    int kaInterval;
+    int kaProbes;
+    if (queryKeepAlive(kaTime, kaInterval, kaProbes))//query OpenLDAP per connection tcp-keepalive settings
+    {
+        if (kaTime != -1 || kaInterval != -1 || kaProbes != -1)
+            DBGLOG("queryKeepAlive returns time=%d, intvl=%d, probes=%d", kaTime, kaInterval, kaProbes);
```

Comment:
ok, but "queryKeepAlive" maybe should be "LDAP tcp keepalive settings:" or something.  I was thinking more like:
```
StringBuffer kaLogStr;

rc = ldap_set_option()
if (rc != LDAP_SUCCESS)
     ERRLOG()
else
     kaLogStr.appendf(" time: %d", kaTime);
```
then at the end of the block:
```
if (!kaLogStr.isEmpty())
    DBGLOG("LDAP tcp keepalive%s", kaLogStr.str());
```